libqtopensesame/misc/sketchpad_canvas.py

```diff
@@ -640,29 +640,55 @@ def line(self, sx, sy, ex, ey, color=None, penwidth=None, add=True):
 		if add:
 			self.addItem(i)
 		return i
+		d = math.sqrt((ey-sy)**2 + (sx-ex)**2)
+		# direction  
+		angle = math.atan2(ey-sy,ex-sx)
+
+
+
+
+	def arrow(self,sx,sy,ex,ey,proportion = 0.7, arrow_width = 30,color = None, 
```

Comment:
Please keep the style consistent with the rest of the code, i.e. no spaces between keyword definitions, respect the 80 character limit, no random empty lines, etc. I know I'm obsessive about this, but it's so little trouble ....